### PR TITLE
feat: Version flag

### LINKED ISSUE #259: Add a version command
Hi,

would it be possible to add a `kubent --version` or `kubent version` command to check the current version?

I'd like to use that command to verify that the installation was correctly. Currently, when running `kubent version` it'll start `kubent` without any error or waning :D

### PR DESCRIPTION
**Pull request by dark0dave**
Closes #259

### CODE REVIEW

**Comment by dark0dave** [comment]
needs #269

File: pkg/config/config.go [hunk 1]
```diff
@@ -36,6 +37,7 @@ func NewFromFlags() (*Config, error) {
 	flag.BoolVarP(&config.Cluster, "cluster", "c", true, "enable Cluster collector")
 	flag.StringVarP(&config.Context, "context", "x", "", "kubeconfig context")
 	flag.BoolVarP(&config.ExitError, "exit-error", "e", false, "exit with non-zero code when issues are found")
+	flag.BoolVarP(&config.KubentVersion, "version", "v", false, "fetch version information of kubent binary")
```
Comment (stepanstipl) [inline pkg/config/config.go:40]: This doesn't match the output added to the README. I would prefer that formulation:
```
prints the version of kubent and exits
```

**Review by stepanstipl — COMMENTED** [review]
Looks good, just align the above. 👍

File: pkg/config/config.go (hunk 1 above)
Comment (dark0dave) [reply pkg/config/config.go]: Thats actually what I had first. Done!

**Review by stepanstipl — APPROVED** [review]
Cool, thx. 👍 🎉

**Pull request merged**